### ISSUE TITLE
Features are not enabled again when all nodes were shutdown

### ISSUE THREAD

**Issue by asias** [scylladb/scylla#4341]
In #4319, user found another issue when trying to replace a node, STREAM_WITH_RPC_STREAM feature is not enabled on the cluster. 

I managed to reproduce with the following procedure. 

```
1. start n1 n2 n3
2. shutdown n1,  n2, n3
3. start n1, n2
4. start n4 to replace n3

n1 and n2 did not enable the rpc streaming features after step 3, so n1 and n2 will use the old streaming to stream to n4. 
```

**Comment by asias** [comment]
The patch on list

[PATCH scylla v4 0/5] Fix empty remote common_features in check_knows_remote_features

can fix this issue.
